botx/bots/mixins/requests/mixin.py

```
@@ -63,20 +63,20 @@ async def call_method(  # noqa: WPS211
             Response for method.
         """
         if not isinstance(method, Token):
-            host = cast(str, credentials.host if credentials else host)
             await _fill_token(
                 self.client,
-                host,
-                cast(UUID, credentials.bot_id if credentials else bot_id),
-                cast(CredentialsSearchProtocol, self).get_cts_by_host(host),
+                cast(CredentialsSearchProtocol, self).get_account_by_bot_id(
+                    credentials.bot_id if credentials else bot_id,  # type: ignore
```

What error was hidden by comment?

---

excepted UUID, got Optional[UUID]